tools/champ_blocs/cb_dissimilarity.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="cb_dissim" name="Dissimilarity" version="@VERSION@" profile = "20.01">
+    <description>indicator</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="cb_requirements">
+        <requirement type="package" version="2.16">r-rmarkdown</requirement>
+        <requirement type="package" version="0.2.4">r-officedown</requirement>
+        <requirement type="package" version="2.6_2">r-vegan</requirement>
+        <requirement type="package" version="1.3.2">r-tidyverse</requirement>
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement>
+    </expand>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/cb_dissimilarity.r'
+            '$fiche_val'
+            '$input_data'
+            '$choice'
+            '$choice_date'
+            '$output_qecb'
+            '$output_eco'
+            '$plots_bf'
+            '$plots_bm'
+            '$plots'
+        ]]>
+    </command>
+    <inputs>
+        <expand macro="cb_input"/>
+        <param name="input_data" type="data" format="data" label="Input champbloc_qecb.csv"/>
+        <param name="choice" type="select" label="Do you have data after the year 2021 ?" display="radio">
+            <option value="N">No</option>
+            <option value="Y">Yes</option>
+        </param>
+        <param name="choice_date" type="integer" value="2021" label="Until when do you have data (write only the YEAR) ?" help="Example write : 2021 "/>
+    </inputs>
+    <outputs>
+        <data name="output_qecb" from_work_dir="qecbnato0.RDS" format="Rdata" label="Clean Rdata"/>
+        <data name="output_eco" from_work_dir="bret_egmp_basq_qecb.RDS" format="Rdata" label="Bret Rdata"/>
+        <collection type="list" name="plots_bf" label="Bloc fixe et bloc mobile face sup plot">
+            <discover_datasets pattern="bm(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+        </collection>
+        <collection type="list" name="plots_bm" label="Bloc mobile face sup et face inf plot">
+            <discover_datasets pattern="fs(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+        </collection><!--&lowbar; pour underscore-->
+        <collection type="list" name="plots" label="Distance plot">
+            <discover_datasets pattern="distance(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
```

Comment:
please add num_expct_output https://docs.galaxyproject.org/en/latest/dev/schema.html#id92

And test if your collections are working.